File: algorithm/src/main/java/baekjoon/p05363/Main.java
package baekjoon.p05363;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();

        while (n-- > 0) {
            System.out.println(solve(sc.nextLine()));
        }
    }

    private static String solve(String input) {
        StringBuilder sb = new StringBuilder();
        String[] inputs = input.split(" ");

        for (int i = 2; i < inputs.length; i++) {
            sb.append(inputs[i]);
            sb.append(" ");
        }

        for (int i = 0; i < 2; i++) {
            sb.append(inputs[i]);
            sb.append(" ");
        }

        return sb.toString();
    }

}

File: algorithm/src/main/java/baekjoon/p05363/Main.kt
package baekjoon.p05363

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        println(solve(readLine()!!))
    }
}

private fun solve(input: String): String {
    val inputs = input.split(" ")
    return buildString {
        for (i in 2 until inputs.size) {
            append(inputs[i])
            append(" ")
        }

        for (i in 0 until 2) {
            append(inputs[i])
            append(" ")
        }
    }
}